### PR TITLE
Upgrade to spire 1.11.0

### PR DESCRIPTION
**Pull request by kfox1111**

### CODE REVIEW

File: charts/spire/charts/spire-agent/values.yaml [hunk 1]
```diff
@@ -227,8 +227,8 @@ workloadAttestors:
         fileName: ""
     ## @param workloadAttestors.k8s.disableContainerSelectors Set to true if using holdApplicationUntilProxyStarts in Istio
     disableContainerSelectors: false
-    ## @param workloadAttestors.k8s.useNewContainerLocator If true, enables the new container locator algorithm that has support for cgroups v2. Defaults to false
-    useNewContainerLocator: false
+    ## @param workloadAttestors.k8s.useNewContainerLocator If true, enables the new container locator algorithm that has support for cgroups v2. Defaults to true
+    useNewContainerLocator: true
```
Comment (faisal-memon) [inline charts/spire/charts/spire-agent/values.yaml:231]: Should we note this change in the upgrade notes?

Comment (kfox1111) [reply charts/spire/charts/spire-agent/values.yaml:231]: its there already

Comment (faisal-memon) [reply charts/spire/charts/spire-agent/values.yaml:231]: I dont see it in there. Just a simple `workloadAttestors.k8s.useNewContainerLocator is now enabled by default.` should do it.

Comment (kfox1111) [reply charts/spire/charts/spire-agent/values.yaml:231]: https://github.com/spiffe/helm-charts-hardened/blob/spire-1.11/charts/spire/README.md?plain=1#L95

**Review by faisal-memon — APPROVED** [review]

**Pull request merged**